pkg/backend/subscriptions.go

```diff
@@ -72,6 +74,16 @@ func (sb *subscriptionBackend) handle(ctx context.Context, log *logrus.Entry, do
 		return sb.endLease(ctx, stop, doc, false, false)
 	}
 
+	defer sb.m.EmitFloat("backend.duration",
```

Comment:
same comments as above
PLUS
can this go in try() to be the same as openshiftclusterbackend?